client/Dockerfile

```diff
@@ -1,40 +1,51 @@
 # Install dependencies only when needed
-FROM node:16-alpine AS deps
+FROM node:17-alpine AS deps
+
 # Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
 RUN apk add --no-cache libc6-compat
+
 WORKDIR /app
+
 COPY package.json yarn.lock ./
+
 RUN yarn install --frozen-lockfile
 
 # If using npm with a `package-lock.json` comment out above and use below instead
 # COPY package.json package-lock.json ./ 
 # RUN npm ci
 
 # Rebuild the source code only when needed
-FROM node:16-alpine AS builder
+FROM node:17-alpine AS builder
+
 WORKDIR /app
+
 COPY --from=deps /app/node_modules ./node_modules
 COPY . .
 
 # Next.js collects completely anonymous telemetry data about general usage.
 # Learn more here: https://nextjs.org/telemetry
 # Uncomment the following line in case you want to disable telemetry during the build.
```

Comment:
you can remove these comments

Reply:
resolved here: https://github.com/MaxLeiter/Drift/pull/75/commits/fa740a9b31b4f4782dfd1c1e5ba079724fdfcfd0